<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectDate</class>
 <widget class="QWidget" name="SelectDate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f3e8d5;</string>
  </property>
  <widget class="QDateEdit" name="Check_in_Date">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
alternate-background-color: #e6a081;</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>7</month>
     <day>19</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>7</month>
     <day>19</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>2025</year>
     <month>7</month>
     <day>19</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2025</year>
     <month>7</month>
     <day>19</day>
    </date>
   </property>
  </widget>
  <widget class="QDateEdit" name="Check_out_Date">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2001</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1752</year>
     <month>9</month>
     <day>27</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>1752</year>
     <month>9</month>
     <day>27</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2001</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Check In Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Check Out Date</string>
   </property>
  </widget>
  <widget class="QComboBox" name="AdultsBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f3e8d5;
color: black;</string>
   </property>
   <property name="currentText">
    <string>1</string>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Adults</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ChildrenBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>background-color: #f3e8d5;
color: black;</string>
   </property>
   <property name="currentText">
    <string>0</string>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Children</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>214</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  border: 2px transparent;
  border-radius: 10px;
  padding: 0 8px;
  background: #bc663f;
 color: White
}
QPushButton:hover {
background-color: #e6a081; 
color: White;              
}</string>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
